v4-client-py/v4_client_py/clients/helpers/chain_helpers.py

```diff
@@ -25,6 +26,41 @@ class OrderExecution(Flag):
     POST_ONLY = auto()  # Post-only
     FOK = auto()        # Fill or Kill
 
+# Enums to use in order proto fields. Use proto generated fields once that's fixed.
+class Order_TimeInForce(Flag):
```

Comment:
Suggestion: note this has to exactly match the [`TimeInForce` field](https://github.com/dydxprotocol/v4-chain/blob/efca789d66c5c424d50accbc8156d2c7031cc5ab/proto/dydxprotocol/clob/order.proto#L159).